kobo/settings/base.py

```diff
@@ -1076,9 +1076,23 @@ def dj_stripe_request_callback_method():
     STRIPE_LIVE_SECRET_KEY = env.str('STRIPE_LIVE_SECRET_KEY', None)
     STRIPE_TEST_SECRET_KEY = env.str('STRIPE_TEST_SECRET_KEY', None)
     DJSTRIPE_WEBHOOK_SECRET = env.str('DJSTRIPE_WEBHOOK_SECRET', None)
+    DJSTRIPE_WEBHOOK_URL = env.str('DJSTRIPE_WEBHOOK_URL', None)
+    DJSTRIPE_UUID = env.str('DJSTRIPE_UUID', None)
+    DJSTRIPE_WEBHOOK_ID = env.str('DJSTRIPE_WEBHOOK_ID', None)
     DJSTRIPE_WEBHOOK_VALIDATION = env.str('DJSTRIPE_WEBHOOK_VALIDATION', 'verify_signature')
+    DOMAIN_NAME = env.str('DOMAIN_NAME', '')
+
 STRIPE_PUBLIC_KEY = STRIPE_LIVE_PUBLIC_KEY if STRIPE_LIVE_MODE else STRIPE_TEST_PUBLIC_KEY
 
+"""SSO Configuration"""
+
+GOOGLE_AUTH_ENABLED = env.bool('GOOGLE_AUTH_ENABLED', False)
+if GOOGLE_AUTH_ENABLED:
+    GOOGLE_CLIENT_ID = env.str('GOOGLE_CLIENT_ID', None)
+    GOOGLE_CLIENT_SECRET = env.str('GOOGLE_CLIENT_SECRET', None)
+    SOCIAL_APP_NAME = env.str('SOCIAL_APP_NAME', None)
+    SOCIAL_APP_PROVIDER_ID = env.str('SOCIAL_APP_PROVIDER_ID', None)
+
```

Comment:
By reading the scripts I suspect that these env variables are required only for the script, at the time of running the script? If that's the case, I suggest to pass these variables to the script via args instead of env variables to make that explicit and more convenient to use, as well as not clutter the env variables that are required for every time kpi starts.